spec/rubocop/cop/rspec/example_wording_spec.rb

```diff
@@ -351,5 +351,95 @@
         end
       RUBY
     end
+
+    context "when message includes `'` in `'...'`" do
```

Comment:
I think these examples are in the wrong context (`'when `DisallowedExamples: Workz`'`).